sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -870,11 +870,16 @@ Accept wildcard characters: False
 
 ### -DisabledWebPartIds
 
-Allows administrators prevent certain, specific web parts from being added to pages or rendering on pages on which they were previously added. Currently, only Amazon Kindle, YouTube and Twitter web parts can be disabled in such a manner.
+Allows administrators to prevent certain web parts from being added to pages or rendering on pages on which they were previously added. Currently, only the folling web parts can be disabled in such a manner:
```

Comment:
```suggestion
Allows administrators to prevent certain web parts from being added to pages or rendering on pages on which they were previously added. Currently, only the following web parts can be disabled in such a manner:
```